### PR TITLE
CSCFAIRMETA-782: [ADD] Enable separate login for old proxy

### PR DESCRIPTION
**Pull request by MattiasLevlin**
- Enables separate login through Etsin/Qvain, through saving the same session for both
- Thanks a lot to @tahme for help with this PR ("ghost programmer" in this case)

### CODE REVIEW

**Comment by MattiasLevlin** [comment]
Requires https://github.com/CSCfi/etsin-ops/pull/38

File: etsin_finder/frontend/js/components/general/navigation/index.jsx [hunk 1]
```diff
@@ -35,7 +36,7 @@ const Navi = ({ routes }) => {
         }
       }}
       onClick={() => {
-        announce(translate('changepage', { page: translate(route.label) }))
+        Accessibility.announce(translate('changepage', { page: translate(route.label) }))
```
Comment (tahme) [inline etsin_finder/frontend/js/components/general/navigation/index.jsx:39]: I noticed the same bug, it's because how `this` works in javascript. It's better not to import the stores directly. I'd revert the changes in this file and edit the constructor `accessibility.js` like this:
```
class Accessibility {
  constructor() {
    makeObservable(this)
    this.announce = this.announce.bind(this)
  }
```

File: etsin_finder/frontend/js/components/qvain/header/index.jsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ const QvainHeader = () => {
       <Right>
         <Settings helpUrl={helpUrl} loginThroughService="qvain" />
         <DesktopOnly>
-          <LoginButton loginThroughService="etsin" />
+          <LoginButton loginThroughService="qvain" />
```
Comment (tahme) [inline etsin_finder/frontend/js/components/qvain/header/index.jsx:58]: Good catch.

File: etsin_finder/frontend/js/components/general/navigation/index.jsx (hunk 1 above)
Comment (MattiasLevlin) [reply etsin_finder/frontend/js/components/general/navigation/index.jsx]: Great suggestion. Resolved with 3ef8c9e and 15761bf

**Review by tahme — APPROVED** [review]
Looks good now and RelayState also works. :+1:
